argo-cd-apps/base/tekton-ci.yaml

```diff
@@ -14,6 +14,13 @@ spec:
     namespace: tekton-ci
     server: https://kubernetes.default.svc
 
+
+  ignoreDifferences:
+  - group: ""
+    kind: "ServiceAccount"
```

Comment:
Should we set `name` here as well since this is only applicable to the `pipeline` service account anyways?